### PR TITLE
AUTOMATIC DEPLOYMENT: As a developer, I want to have continuous integration set up, so that I can have a nicer workflow

### PR DESCRIPTION
**Pull request by Reinkaos**
**Acceptance Criteria**

- The docker build/push is done in a pipeline
- There's something to do to deploy the docker image (github actions, travis?)
- The process is documented in the README if needed

### CODE REVIEW

File: .github/workflows/production_build_and_deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Production build and Deploy
+
+on:
+  push:
+    branches: master
```
Comment (paul-mesnilgrente) [inline .github/workflows/production_build_and_deploy.yaml:5]: Could you do on release maybe? It shouldn't change anything to the rest of the yaml

Comment (Reinkaos) [reply .github/workflows/production_build_and_deploy.yaml:5]: Yeah sure, do you mean on tag?

**Comment by Reinkaos** [comment]
Sure I'll have a look at that airbrake deploy :)

File: .github/workflows/production_build_and_deploy.yaml (hunk 1 above)
Comment (paul-mesnilgrente) [reply .github/workflows/production_build_and_deploy.yaml]: There should be on push release I believe. That's what I had done on combiner

**Comment by paul-mesnilgrente** [comment]
The README has some information around airbrake:deploy

File: .github/workflows/production_build_and_deploy.yaml (hunk 1 above)
Comment (Reinkaos) [reply .github/workflows/production_build_and_deploy.yaml]: Ok cool I'll give that a go

**Pull request merged**